  Extension.cpp
c:\users\cloud\source\repos\a3db\a3db\src\a3db\extension.cpp(70): warning C4267: 'initializing': conversion from 'size_t' to 'unsigned int', possible loss of data
c:\users\cloud\source\repos\a3db\a3db\src\a3db\extension.cpp(184): warning C4267: 'initializing': conversion from 'size_t' to 'int', possible loss of data
c:\users\cloud\source\repos\a3db\a3db\src\a3db\safequeue.hpp(63): warning C4267: 'initializing': conversion from 'size_t' to 'int', possible loss of data
  c:\users\cloud\source\repos\a3db\a3db\src\a3db\safequeue.hpp(55): note: while compiling class template member function 'bool SafeQueue<U>::try_pop_results(std::vector<Result,std::allocator<_Ty>> &,unsigned int,unsigned int)'
          with
          [
              U=Result,
              _Ty=Result
          ]
  c:\users\cloud\source\repos\a3db\a3db\src\a3db\processor.hpp(64): note: see reference to function template instantiation 'bool SafeQueue<U>::try_pop_results(std::vector<Result,std::allocator<_Ty>> &,unsigned int,unsigned int)' being compiled
          with
          [
              U=Result,
              _Ty=Result
          ]
  c:\users\cloud\source\repos\a3db\a3db\src\a3db\processor.hpp(96): note: see reference to class template instantiation 'SafeQueue<U>' being compiled
          with
          [
              U=Result
          ]
  c:\users\cloud\source\repos\a3db\a3db\src\a3db\extension.cpp(47): note: see reference to class template instantiation 'Processor<Workload,Result>' being compiled
     Creating library C:\Users\Cloud\Source\Repos\A3DB\A3DB\src\x64\Debug\A3DB.lib and object C:\Users\Cloud\Source\Repos\A3DB\A3DB\src\x64\Debug\A3DB.exp
  A3DB.vcxproj -> C:\Users\Cloud\Source\Repos\A3DB\A3DB\src\x64\Debug\A3DB.dll
  A3DB.vcxproj -> C:\Users\Cloud\Source\Repos\A3DB\A3DB\src\x64\Debug\A3DB.pdb (Partial PDB)
